From: Simon Eisenmann <simon@struktur.de>
Date: Mon, 26 Oct 2015 14:18:05 +0100
Subject: translate: disable fetching of languages from server

Prevents Chromium's Google Translate features from fetching languages on Google's servers.

Initial Commit: https://github.com/uazo/cromite/commit/fc14caf48a4700f0d8cf5175b582f31a5a739c82#diff-145ccb8407fe599ebbb5820e6dcb3ccfbf2327a67dd8bd3d54c7719ae64184c7 (Inherited from Bromite)
Category: Privacy
License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 components/translate/core/browser/translate_language_list.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/components/translate/core/browser/translate_language_list.cc b/components/translate/core/browser/translate_language_list.cc
--- a/components/translate/core/browser/translate_language_list.cc
+++ b/components/translate/core/browser/translate_language_list.cc
@@ -393,6 +393,7 @@ GURL TranslateLanguageList::TranslateLanguageUrl() {
 }
 
 void TranslateLanguageList::RequestLanguageList() {
+#if 0
   // If resource requests are not allowed, we'll get a callback when they are.
   if (!resource_requests_allowed_) {
     request_pending_ = true;
@@ -422,6 +423,7 @@ void TranslateLanguageList::RequestLanguageList() {
     if (!result)
       NotifyEvent(__LINE__, "Request is omitted due to retry limit");
   }
+#endif
 }
 
 void TranslateLanguageList::SetResourceRequestsAllowed(bool allowed) {
--
